x-pack/plugins/security_solution/public/app/home/template_wrapper/index.tsx

```diff
@@ -59,6 +61,17 @@ const StyledKibanaPageTemplate = styled(KibanaPageTemplate)<{
       }
     }
   `}
+
+  // If the policy settings bottom bar is visible add padding to the navigation
+  ${({ $isPolicySettingsVisible }) =>
+    $isPolicySettingsVisible &&
+    `
+    @media (min-width: 768px) {
+      .kbnPageTemplateSolutionNav {
+        padding-bottom: ${gutterTimeline};
+      }
+    }
+  `}
```

Comment:
If you want to create another conditional here then I'll suggest that you extract this duplicated bit into a CSS mixin that you can import here.